### PR TITLE
Export as just orderHeader for API spec

### PR DESCRIPTION
**Pull request by Oxymoron290**

### CODE REVIEW

File: AllfleXML/FlexOrder/FlexOrder.cs [hunk 1]
```diff
@@ -54,7 +57,15 @@ public static Document Import(XDocument document)
 
         public static XDocument Export(this OrderHeader order)
         {
-            return Export(new Document { OrderHeaders = new List<OrderHeader> { order } });
+            var result = new XDocument();
```
Comment (TexasGael) [inline AllfleXML/FlexOrder/FlexOrder.cs:60]: Dont we want it to do both?  Export as just an Order and also as a Document?

Comment (Oxymoron290) [reply AllfleXML/FlexOrder/FlexOrder.cs:60]: Because the api.allflexusa.com did not follow this specification, this is a nessessity

Comment (Oxymoron290) [reply AllfleXML/FlexOrder/FlexOrder.cs:60]: you can't post a document to the api, you can only post an orderHeader

Comment (TexasGael) [reply AllfleXML/FlexOrder/FlexOrder.cs:60]: ah, wait, I see.   We do still have the ability to Export a document if need be.

**Review by TexasGael — APPROVED** [review]

**Pull request merged**